### PR TITLE
Feat/result animation

### PR DESCRIPTION
**Pull request by elenamarinaki**
- results component which render info dynamically based on the different ranges that come from the API
- animation for the results block
- splash page with runner animation
- placeholder for the map component
- smooth scroll behaviour
- general styling

### CODE REVIEW

File: src/components/Result.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+const CircleOut = styled.div`
+  background-color: white;
+  border-radius: 100%;
+  width: 50px;
+  height: 50%;
+`;
+
+const Circle = styled.div`
+  background-color: ${props => props.color}78;
```
Comment (sandiskolarczyk) [inline src/components/Result.js:36]: 78?

**Review by adrianHards — APPROVED** [review]
fantastic work guys!

File: public/index.html [hunk 2]
```diff
@@ -24,7 +31,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Clean Run</title>
```
Comment (11ma) [inline public/index.html]: Suggestion: Clean runnings

**Review by sandiskolarczyk — APPROVED** [review]
Awesome job! ⭐️

**Review by 11ma — COMMENTED** [review]
great job

**Pull request merged**